src/main/java/seedu/address/model/service/Service.java

```diff
@@ -78,6 +77,14 @@ public Service(int id, int vehicleId) {
         this(id, vehicleId, 7, "");
```

Comment:
This one not your fault but I think we should change to `this(id, vehicleId, DEFAULT_SEVEN_DAYS, "");` where `static int DEFAULT_SEVEN_DAYS = 7`

Reply:
Will do